Touch Bar Simulator/TouchBarWindow.swift

```diff
@@ -45,6 +66,83 @@ final class TouchBarWindow: NSPanel {
 		}
 	}
 
+	var dockBehaviorTimer: Timer?
+
+	func startDockBehaviorTimer() {
+		stopDockBehaviorTimer()
+		// Throttles the function to only execute every 1/10 second.
+		// Without any throttling the function would be very expensive and use an average of 7% CPU (on my machine) which is too much IMO.
```

Comment:
What was it using before the throttling? That's unclear from the comment.

Reply:
I forgot to update the comment. I used the following before:
`NSEvent.addGlobalMonitorForEvents(matching: .mouseMoved) { event in }`
Anyway, there is no a reason to run the function more frequently, because the user will not notice and it saves a bit CPU.
I have just removed the comment for now.